src/plugins/intel_cpu/tests/functional/single_layer_tests/ctc_greedy_decoder_seq_len.cpp

```diff
@@ -107,7 +97,11 @@ class CTCGreedyDecoderSeqLenLayerCPUTest : public testing::WithParamInterface<CT
                 targetStaticShapes.push_back({{N, T, C}, {N}, {1}});
         }
 
-        auto params = makeDynamicParams(inType, inputDynamicShapes);
+        ov::ParameterVector params;
+        for (size_t i = 0; i < inType.size(); i++) {
```

Comment:
check `inType.size() == inputDynamicShapes.size()` is required, otherwise it will lead to segfault